src/support/colors.cpp

```diff
@@ -19,24 +19,39 @@
 #include <cstdlib>
 #include <ostream>
 
-#if defined(__linux__) || defined(__APPLE__)
-# define CAN_HAZ_COLOR 1
-# include <unistd.h>
-#endif
-
 namespace {
 bool colors_disabled = false;
 }  // anonymous namespace
 
 void Colors::disable() { colors_disabled = true; }
 
+#if defined(__linux__) || defined(__APPLE__)
+#include <unistd.h>
+
 void Colors::outputColorCode(std::ostream& stream, const char* colorCode) {
-#if defined(CAN_HAZ_COLOR)
   const static bool has_color = []() {
     return (getenv("COLORS") && getenv("COLORS")[0] == '1') ||  // forced
            (isatty(STDOUT_FILENO) &&
             (!getenv("COLORS") || getenv("COLORS")[0] != '0'));  // implicit
   }();
   if (has_color && !colors_disabled) stream << colorCode;
+}
 #endif
+
```

Comment:
the `endif` and `if` can be merged into an `elif`, since they are mutually exclusive.